renku/api/repository.py

```diff
@@ -375,3 +375,13 @@ def init_repository(self, name=None, force=False):
             metadata.updated = datetime.datetime.utcnow()
 
         return str(path)
+
+    def is_untracked(self, path):
```

Comment:
Let's make this a normal function and put it to _git.py. That way I can reuse it in my check.

Reply:
👍

Reply:
actually I realized I'm not using it anywhere anymore so I removed it.